docs/roadmap.md

```diff
@@ -0,0 +1,36 @@
+# `rust-bitcoin` repository roadmap
+
+* ~Release `units v1.0.0-rc0`~
+* Release `primitives v1.0.0-rc0`
+  * Implement script tagging
+  * Add support for consensus encoding to `primitives`
+* Release `bitcoin v0.33.0-rc0`
+* Split out an address crate
+* BIP-32 and BIP-380
+* PSBTv2
+
+## RC cycle
+
+Six month RC cycle for `units` and `primitives` (implies `bitcoin`). Then start cutting the `1.0`
+releases and `bitcoin v0.33`.
+
+## Split out an address crate
```

Comment:
In a3c03adf407483240417df158db7a6230e9cb45e:

Rather than using a subsection here can you make this a bulletpoint in the roadmap list above which links to another (stub) document with more detail? Same with the BIP32/380 crate and psbt below.